### PR TITLE
CI - add workflows lint check

### PR DESCRIPTION
**Pull request by guidoiaquinti**
## Description
Fixes https://github.com/PostHog/charts-clickhouse/issues/259

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## How has this been tested?
CI should be passing

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: .github/disabled-test-digitalocean-1-click-install.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # #
 # # Unfortunately we can test this scenario only after merging to 'main' as we don't have
 # # control over the branch used by the DigitalOcean's marketplace 1-click install.
-# #
```
Comment (guidoiaquinti) [inline .github/disabled-test-digitalocean-1-click-install.yaml:14]: I've renamed and moved this disabled workflow to don't make CI complaining

File: .github/workflows/test-lint-github-worklows.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: Lint test
+
+on:
+  workflow_dispatch:
```
Comment (macobo) [inline .github/workflows/test-lint-github-worklows.yaml:4]: Do we need this realistically?

**Review by macobo — APPROVED** [review]
Very meta

File: .github/workflows/test-lint-github-worklows.yaml (hunk 2 above)
Comment (guidoiaquinti) [reply .github/workflows/test-lint-github-worklows.yaml:4]: Unlikely, but I don't think it can cause any harm keeping it

**Pull request merged**